mainpage.dox

```diff
@@ -19,8 +19,8 @@
  *
  *  eBPF programs can also call helper APIs to do additional processing.  There are two types of helpers:
  *  <ul>
- *   <li>Helpers defined in bpf_helpers.h which are accessible to all program types.
- *   <li>Helpers specific to a given program type and declared in other headers.
+ *   <li>Helpers that are accessible to all program types, defined in bpf_helper_defs.h.
+ *   <li>Helpers specific to a given program type and declared in other headers, such as ebpf_nethooks.h.
```

Comment:
`given network related program type` ?

Reply:
no, that's just for ebpf_nethooks.h which is in the "such as" phrase at the end.  The earlier part of the sentence is more general and worded to apply to any such header.